pytest_fixtures/api_fixtures.py

```diff
@@ -46,6 +46,12 @@ def module_org():
     return entities.Organization().create()
 
 
+@pytest.fixture(scope='module')
+def orgs(request):
```

Comment:
To maintain consistency in the fixture naming, I would go with `module_orgs`

Reply:
git blame suggests that is not part of this PR:
7737854b1 robottelo/api/entity_fixtures.py (Jitendra Yejare 2020-02-17 15:00:23 +0530  45) def module_org():

It creates one org and its widely used in that file.

Reply:
Miscommunication.

The current fixtures for organization creation are called `module_org` for the module scoped fixture, and `default_org` for the session scoped fixture - as `default_*` was widely used across the test suite for session scoped entities.

Entity fixtures (with some early adopter exceptions that should be cleaned up) follow a naming pattern of `<scope>_<entity>` - `module_org`, `module_location`, `function_role`, etc.

To be consistent with existing fixture names, and this pattern, your new fixture should be called `module_orgs`, a pluralization of the current `module_org` fixture.

Reply:
Think I actually misread your response. You didn't intend to define this fixture, and haven't committed it to your branch?

You need to rebase and clean up your commits in that case, if you don't intend to add this fixture in this PR.

Reply:
oh dear, this bit should not have been committed:
    +@pytest.fixture(scope='module')
    +def orgs(request):
    +    num = getattr(request, 'param', 2)
    +    return [entities.Organization().create() for i in range(num)]

It was an idea that I did not use in the end. I will remove and test again.